src/panels/lovelace/cards/hui-sensor-card.ts

```diff
@@ -175,6 +192,7 @@ class HuiSensorCard extends LitElement implements LovelaceCard {
       detail: 1,
       theme: "default",
       hours_to_show: 24,
+      fill: true,
```

Comment:
Should we make this configurable or should we just always set it to `true` ?

Reply:
I'm down to make it always true. I think it looks way better like this

Reply:
Alright. Let's drop it .